docs/blog/2018-10-06-deploying-gatsby-to-azure/index.md

```diff
@@ -0,0 +1,154 @@
+
+```bash
+$ node -v
+$ v10.9.0
+```
+
+Install VS Code from the [VS Code website](https://code.visualstudio.com/).
+
+Next, we can install the important part! [Gatsby](https://gatsbyjs.org/) from your terminal, run:
+
+```bash
+$ npm install --global gatsby-cli
++ gatsby-cli@2.4.2
+updated 1 package in 4.868s
+```
+
+## Creating your blog
+
+Next you will create the blog from the handy [Gatsby Starter Blog](https://github.com/gatsbyjs/gatsby-starter-blog).
+
+> If you already have a site you want to deploy, skip forwards to [Setting up Azure](#setting-up-azure)
+
+From your terminal run
+
+```bash
+$ gatsby new azure-blog https://github.com/gatsbyjs/gatsby-starter-blog
+
+info Creating new site from git: https://github.com/gatsbyjs/gatsby-starter-blog.git
+Cloning into 'azure-blog'...
+remote: Enumerating objects: 1, done.
+remote: Counting objects: 100% (1/1), done.
+remote: Total 1139 (delta 0), reused 0 (delta 0), pack-reused 1138
+Receiving objects: 100% (1139/1139), 2.14 MiB | 2.95 MiB/s, done.
+Resolving deltas: 100% (636/636), done.
+success Created starter directory layout
+info Installing packages...
+...
+```
+
+You can now view your website by running
+
+```bash
+$ cd azure-blog
+$ gatsby develop
+```
+
+You should now see your blog in the flesh!
+
+![Gatsby blog starter](blogstarter.png)
+
+## Setting up Azure
+
+Microsoft Azure is a cloud hosting provider which offers a wide range of tools and services. Today you are going to use Azure Storage to host our static Gatsby blog.
+
+> If you don't have an Azure account, [sign up today](https://azure.microsoft.com/en-us/free/) for a free 30 day account with $200 in Azure credits.
+
+As part of this tutorial you will:
+
+- Create an `Azure Storage Account` for our blog
+- Install the `Azure Storage` VS Code extension
+- Create a production build of your blog
+- Deploy your blog to Azure
+
+### Create the Azure Storage Account
+
+Login to the [Azure Web Portal]() and select `+ Add Resource`, find and select `Storage Account` in the list.
+
+Fill in the form by selecting your subscription, creating a new resource group if you don't have one and giving your storage account a name and region. Your form should look something like this.
+
+![Azure Storage Account Form](createstorageaccount.png)
+
+Click on `review + create` then `create`, after a few seconds, your account will be created.
+
+Next, click on `Go to resource`
+
+![Azure storage account created](createstorageaccountcomplete.png)
+
+### Setup your account for static site usage
+
+Find the `Static Website (preview)` link in the left hand panel and select it.
+
+Change the option to `enabled`, set a default document name of `index.html` and hit `Save`.
+
+![Set default document](setdefaultdocument.png)
+
+> Note down the `Primary Endpoint` that azure gives you, that will be the web address that our blog will be deployed (https://gatsbyblogstorage.z35.web.core.windows.net/)
```

Comment:
We should call out that everyone’s address will be different. So maybe a qualifier, like, “Your web address will look something like https://gatsbyblogstorage.z35.web.core.windows.net/”.

Reply:
Good idea